src/tess_asteroids/movingtpf.py

```diff
@@ -479,6 +485,134 @@ def _create_threshold_mask(
 
         return aperture_mask
 
+    def _ellipse_aperture(
+        self,
+        R: float = 3.0,
+        smooth: bool = True,
+        ellipse_params: bool = False,
```

Comment:
```suggestion
        return_params: bool = False,
```
I think we should rename this `return_params` because it includes both centroids and ellipse parameters.